roachtest: liquibase failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20919530?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20919530?buildTab=artifacts#/liquibase) on release-24.3 @ [796c16b5a98ba9dfeb77ca089402004cd69a21c0](https://github.com/cockroachdb/cockroach/commits/796c16b5a98ba9dfeb77ca089402004cd69a21c0):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v24.3.25-dev-796c16b5a98ba9dfeb77ca089402004cd69a21c0
Tests run against liquibase 91c42e7644c9db5cc86f6d013a33c43900646e2f
50 Total Tests Run
47 tests passed
3 tests failed
15 tests skipped
0 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ChangeObjectTests.apply addCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply createPackage against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply dropCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20919530/liquibase/1767249737306/1767250043984)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58303

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20920665?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20920665?buildTab=artifacts#/liquibase) on release-24.3 @ [5a3d5794765d75e4b775df3f6c4835774a8d0cd8](https://github.com/cockroachdb/cockroach/commits/5a3d5794765d75e4b775df3f6c4835774a8d0cd8):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v24.3.25-dev-5a3d5794765d75e4b775df3f6c4835774a8d0cd8
Tests run against liquibase 91c42e7644c9db5cc86f6d013a33c43900646e2f
50 Total Tests Run
47 tests passed
3 tests failed
15 tests skipped
0 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ChangeObjectTests.apply addCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply createPackage against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply dropCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20920665/liquibase/1767342742277/1767343109123)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #160376 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-26.1 release-blocker]
- #160373 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2 release-blocker]
- #160372 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2.11-rc release-blocker]
- #160371 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4.3-rc release-blocker]
- #160367 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3.7-rc release-blocker]
- #160364 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
- #160362 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4 release-blocker]
- #160360 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3 release-blocker]
- #160358 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.3.25-rc release-blocker]
- #160355 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20923773?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20923773?buildTab=artifacts#/liquibase) on release-24.3 @ [60b41ceffef285d73a0dfe9b93264e33a2dc400a](https://github.com/cockroachdb/cockroach/commits/60b41ceffef285d73a0dfe9b93264e33a2dc400a):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v24.3.25-dev-60b41ceffef285d73a0dfe9b93264e33a2dc400a
Tests run against liquibase 91c42e7644c9db5cc86f6d013a33c43900646e2f
50 Total Tests Run
47 tests passed
3 tests failed
15 tests skipped
0 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ChangeObjectTests.apply createPackage against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply dropCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply addCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>runtimeAssertionsBuild=false</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20923773/liquibase/1767432788434/1767433085454)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #160376 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-26.1 release-blocker]
- #160373 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2 release-blocker]
- #160372 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2.11-rc release-blocker]
- #160371 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4.3-rc release-blocker]
- #160367 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3.7-rc release-blocker]
- #160364 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-master release-blocker]
- #160362 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4 release-blocker]
- #160360 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3 release-blocker]
- #160358 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.3.25-rc release-blocker]
- #160355 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20924581?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/20924581?buildTab=artifacts#/liquibase) on release-24.3 @ [60b41ceffef285d73a0dfe9b93264e33a2dc400a](https://github.com/cockroachdb/cockroach/commits/60b41ceffef285d73a0dfe9b93264e33a2dc400a):


```
(orm_helpers.go:224).summarizeFailed: 
Tests run on Cockroach v24.3.25-dev-60b41ceffef285d73a0dfe9b93264e33a2dc400a
Tests run against liquibase 91c42e7644c9db5cc86f6d013a33c43900646e2f
50 Total Tests Run
47 tests passed
3 tests failed
15 tests skipped
0 tests ignored
0 tests passed unexpectedly
3 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ChangeObjectTests.apply dropCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply createPackage against cockroachdb 24.1 - unknown (unexpected)
--- FAIL: ChangeObjectTests.apply addCheckConstraint against cockroachdb 24.1 - unknown (unexpected)
For a full summary look at the liquibase artifacts 
An updated blocklist (liquibaseBlocklist) is available in the artifacts' liquibase log
test artifacts and logs in: /artifacts/liquibase/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-20924581/liquibase/1767511450492/1767511759909)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #160376 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-26.1]
- #160373 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2]
- #160372 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.2.11-rc]
- #160371 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4.3-rc]
- #160367 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3.7-rc]
- #160364 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-master]
- #160362 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.4]
- #160360 roachtest: liquibase failed [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-release-25.3]
- #160358 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.3.25-rc]
- #160355 roachtest: liquibase failed [C-test-failure O-roachtest O-robot T-sql-foundations branch-release-24.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/cmd/roachtest/tests/liquibase_blocklist.go
+++ b/pkg/cmd/roachtest/tests/liquibase_blocklist.go
@@ -8,7 +8,7 @@ package tests
 var liquibaseBlocklist = blocklist{}
 
 var liquibaseIgnorelist = blocklist{
-	`liquibase.harness.change.ChangeObjectTests.apply addCheckConstraint against cockroachdb 24.1`:  "requires enterprise version of liquibase",
-	`liquibase.harness.change.ChangeObjectTests.apply createPackage against cockroachdb 24.1`:       "requires enterprise version of liquibase",
-	`liquibase.harness.change.ChangeObjectTests.apply dropCheckConstraint against cockroachdb 24.1`: "requires enterprise version of liquibase",
+	`ChangeObjectTests.apply addCheckConstraint against cockroachdb 24.1`:  "requires enterprise version of liquibase",
+	`ChangeObjectTests.apply createPackage against cockroachdb 24.1`:       "requires enterprise version of liquibase",
+	`ChangeObjectTests.apply dropCheckConstraint against cockroachdb 24.1`: "requires enterprise version of liquibase",
 }

